CMakeLists.txt

```diff
@@ -337,7 +337,9 @@ IF(NOT CYCLUS_DOC_ONLY)
 
         # make sure we know about having python
         add_definitions(-DCYCLUS_WITH_PYTHON)
-        set(LIBS ${LIBS} ${PYTHON_LIBRARIES})
+        IF (NOT APPLE)
+            set(LIBS ${LIBS} ${PYTHON_LIBRARIES})
+        ENDIF (!APPLE)
```

Comment:
Is there a reason you open the if statement with ```NOT APPLE``` but end it with ```!APPLE``` ? Does it matter if they exactly match?

Reply:
fair point fixing it